emg3d/electrodes.py

```diff
@@ -560,11 +560,12 @@ class Receiver(Wire):
     # Add relative to attributes which have to be serialized.
     _serialize = {'relative'} | Wire._serialize
```

Comment:
I have to test, but I think you need to add 'data_type' to `_serialize`.

Reply:
Currenty, if you initiate a receiver as
```
r = emg3d.RxElectricPoint((0, 0, 0, 0, 0), data_type='amp')
```
it will be amplitude, but
```
emg3d.RxElectricPoint.from_dict(r.to_dict())
```
will create a complex receiver. That is why we need it in serialize

Reply:
ah, cool. did not realize that.